### DISCUSSION TITLE
Material property 'diffH2', requested by 'H2_diffusion' is not defined on block 1

### DISCUSSION THREAD

**Discussion by JaeHyun-Nam** [shannon-lab/zapdos#97] [Q&A Getting Started] [answered]
Hello, thank you for the previous question.

I have an issue while adding "H2" to "mean_en.i" file as a first step of modifying this codes.

I made a variable "H2" and also defined [./H2_time_deriv] and [./H2_diffusion]

Also I modified Water.h and Water.c files to include "diffH2".

But the error ""Material property 'diffH2', requested by 'H2_diffusion' is not defined on block 1"" keep appearing

I'll appreciate it if anyone can guide me.

**Comment by JaeHyun-Nam** [comment, marked as answer]
I solved this problem by 
make -j4 command

**Reply by lindsayad** [reply]
I'm glad you were able to get it figured out!

**Reply by lindsayad** [reply]
I saw a question about adding a reaction in my email, but it looks like you must have edited your post?

**Reply by JaeHyun-Nam** [reply]
Yes, I have edited since I found the presentation of Dr. Davide Curreli who showed the Zapdos-Crane simulation where more reactions are added (and my PI also mentioned this). However, I wonder where I could find the technical notes of Zapdos-Crane coupling...... Do you know where to find this (tutorial or codes)...? I apologize if these are so naive questions.

**Reply by csdechant** [reply]
Zapdos-Crane coupling is automatically set up for the default Zapdos branch. This means that users can use any Crane or Zapdos objects within a one input file, without any additional setup. For documentation, there is the Zapdos_Tutorial file in the `doc` folder. For code examples, there are the test files in the `/tests/crane_action` folder.
